manifests/e/ElijahLopez/MusicCaster/4.90.123/ElijahLopez.MusicCaster.installer.yaml

```diff
@@ -0,0 +1,24 @@
+# Created with YamlCreate.ps1 v2.0.0 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: ElijahLopez.MusicCaster
+PackageVersion: 4.90.123
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: user
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Protocols:
+- https
+- http
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/elibroftw/music-caster/releases/download/v4.90.123/Music.Caster.Setup.exe
```

Comment:
Already 404 👌 I think we need to ask if these are Pre-Release/Beta's..

Reply:
Actually they publish releases and then delete the assets. That's why the automation finds a new release and create a PR, after that they delete the asset and the URL becomes 404 Not Found. Don't know why they do this 🤷🏻‍♂️

Reply:
Developer has this in one of their release notes: `DO NOT DELETE. THIS IS USED by winget-pkgs` for an older version: https://github.com/elibroftw/music-caster/releases/tag/v4.89.0.

Perhaps it will be better to get a response from @elibroftw as to why older releases are kept but not the latest ones.

Reply:
I only keep the major versions of my software. I didn't know this repo was smart enough to make PR's for new releases but if that is so, I will stop deleting old minor versions so that you guys don't have issues.

Reply:
@vedantmgoyal2009 What if you upgraded the automation bot to look for updates when a reviewer uses the /update-pkg command in a PR request? The /update-pkg would check if a new release is available, download it, update the manifest, and push the commit. That way the investigation won't require any input from the author and will save reviewers time as well. Although me not deleting the release anymore will avoid any 404's, even within 6 hours, I had released a new release, and thus this PR, outdated.

Reply:
Currently, the automation looks for updates and creates PR automatically. Since it is not perfect and makes errors sometimes, I prefer creating PRs which are validated by the Moderators till I get to automatic ARP metadata validation.